specification/src/chapter3.adoc

```diff
@@ -146,14 +158,16 @@ memory, execution state, and devices - from other supervisor domains regardless
 of privilege level (below M-mode). Isolation and context switching between
 supervisor domains are managed by M-mode firmware.
 
+A supervisor domain is identified at architecture level by a _supervisor domain id (SDID)_ CSR, managed by M-mode firmware.
```

Comment:
by a supervisor domain ID, managed by M-mode firmware (may not be a CSR itself).

Reply:
Fixed.